### PR TITLE
change links in maim menu

### PR DESCRIPTION
**Pull request by romigos**

### CODE REVIEW

File: app/views/static_pages/index.html.erb [hunk 1]
```diff
@@ -6,39 +6,25 @@
   <div id="menu">
     <ul>
       <li>
-        <a href="#&amp;id=1" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('Image7','','menu_m_services_a.png',1)">
-          <%= image_tag 'menu_m_services.png', width: 221, height: 31, border: 0 %>
-        </a>
+        <a <%= link_to image_tag('menu_m_services.png'), services_path %>
```
Comment (saviokmua) [inline app/views/static_pages/index.html.erb]: @romigos it is not correct. Did you test it on yours end-point?

File: app/views/static_pages/index.html.erb [hunk 2]
```diff
@@ -6,39 +6,25 @@
...
       </li>
       <li>
-        <a href="#&amp;id=2" onmouseout="MM_swapImgRestore()" onmouseover="MM_swapImage('Image8','','menu_m_services_a.png',1)">
-          <%= image_tag 'menu_m_prices.png', width: 221, height: 31, border: 0 %>
-        </a>
+        <a <%= link_to image_tag('menu_m_prices.png'), price_path %>
```
Comment (saviokmua) [inline app/views/static_pages/index.html.erb]: @romigos here and in below rows

**Review by saviokmua — CHANGES_REQUESTED** [review]

File: app/views/static_pages/index.html.erb (hunk 1 above)
Comment (romigos) [reply app/views/static_pages/index.html.erb]: Of course i tested. Everything is working.Give me error that you get or solutions how to make this links.

Comment (romigos) [reply app/views/static_pages/index.html.erb]: sorry i can not do it on heroku but try.

Comment (saviokmua) [reply app/views/static_pages/index.html.erb]: @romigos `<a <%= link_to` it is not correct. please rewrite to `<%= link_to`

**Review by saviokmua — CHANGES_REQUESTED** [review]

**Comment by romigos** [comment]
i make all changes and now i know how to pull changes in another branch. Thanks

**Comment by romigos** [comment]
sorry to push changes.

**Review by saviokmua — APPROVED** [review]

**Pull request merged**